modules/ding_permissions/ding_permissions.module

```diff
@@ -1011,3 +1013,14 @@ function ding_permissions_secure_permissions($role) {
     return $permissions[$role];
   }
 }
+
+/**
+ * Implements hook_page_alter().
```

Comment:
Still would like a comment in the code about this, so we also knowns why it's here in the future.